source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -140,16 +149,46 @@ FilterConfigPerRoute::mergeProcessingMode(const FilterConfigPerRoute& less_speci
                                                     : less_specific.processingMode();
 }
 
+namespace {
+// replace all entries with the same name or append one.
+void mergeHeaderValue(std::vector<envoy::config::core::v3::HeaderValue>& metadata,
+                      const envoy::config::core::v3::HeaderValue& header) {
+  size_t count = 0;
```

Comment:
nit: `bool found = false` .

You only need a boolean flag here to set and check

Reply:
done